code/modules/cargo/packs.dm

```diff
@@ -2271,6 +2370,72 @@
 	crate_name = "coffin"
 	crate_type = /obj/structure/closet/crate/coffin
 
+/datum/supply_pack/misc/hos_headset
```

Comment:
Why should head access keys be purchasable?

Reply:
For lings/Traitors/silicons.